core/src/main/groovy/noe/workspace/WorkspaceAbstract.groovy

```diff
@@ -6,6 +6,7 @@ import noe.common.NoeContext
 import noe.common.utils.Cleaner
 import noe.common.utils.Hudson
 import noe.common.utils.JBFile
+import noe.common.utils.Java
```

Comment:
Is this import required?

Reply:
Not anymore, it was a residue from some earlier version. I thought that idea would prune unneeded imports the same as they add them when needed. Thanks for noticing.